src/fileServices/s3Service.ts

```diff
@@ -190,7 +190,7 @@ export class S3Service implements FileService {
     }
   }
 
-  async getMetadata(key: string) {
+  async getMetadata(key: string): Promise<unknown> {
```

Comment:
Can we have a stricter type than `unknown` here? I don't know if the S3 SDK already has a better type (https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#headObject-property)